docs/monoDrive_home/Lighting API.md

```diff
@@ -0,0 +1,174 @@
+# Lighting API 
+
+The monoDrive lights API allows users to create, configure and control lights in a simulation. A light configuration can be attached to an actor in the simulation and then controlled during the simulation by sending update commands. 
+
+The API provides two commands: the lights configuration command `VehicleLightsConfigCommand_ID` which is used to set up and configure the lights at the beginning of the simulation, and the lights update command `VehicleLightsUpdateCommand_ID` which is used to update the state of the lights (intensity, location, orientation, etc.) during the simulation.
+
+Lights are composed of one or more `ULightObjects` grouped together into a `ULightArray`, which is attached to an actor in the simulation. `ULightObjects` are composed of a `UPointLightComponent` and a `USpotLightComponent`.
+
+
+## Configuration
+The `VehicleLightsConfigCommand_ID` command configures a set of LED arrays and sets the initial state of each LED in the array:
+
```

Comment:
The JSON here is not showing correctly with the JSON tag because of the added comments. Here's how we could add comments and keep the JSON valid. 

```json
{
  "actor_id"	// the id of the actor to attach the lights to
  "lights": [
    {
      "array_id" // id of this light array, used for control
      "position" // x, y, z relative to actor center
      "rotation"// yaw, pitch, roll relative to actor center
      "lights": [
        // array of led configuration
        {
          "led": 0, 	// led index in array
          "position": ,	// x,y,z relative to array
          "rotation":	,// yaw, pitch, roll relative to array
          // backlight (UPointLightComponent) settings
          "backlight_intensity"
          "backlight_color"
          "backlight_attenuation_radius"
          "backlight_source_radius"
          "backlight_soft_source_radius"
          "backlight_source_length"
          "backlight_temperature"
          "backlight_indirect_lighting_intensity"
          "backlight_volumetric_scattering_intensity"

          // spotlight (USpotLightComponent) settings
          "inner_cone_angle"
          "outer_cone_angle"
          "intensity"
          "color"
          "attenuation_radius"
          "temperature"
          "source_radius"
          "soft_source_radius"
          "source_length"
          "indirect_lighting_intensity"
          "volumetric_scattering_intensity"
          "ies_profile"
        },
      ]
    },
  ]
}
```

<img width="501" alt="Screen Shot 2020-11-20 at 2 09 15 PM" src="https://user-images.githubusercontent.com/41026694/99845272-fc3e3e80-2b39-11eb-8595-f137e71f65f3.png">